src/qibolab/runcards/dummy.yml

```diff
@@ -167,13 +167,13 @@ characterization:
             # parameters for single shot classification
             threshold: 0.0
             iq_angle: 0.0
-            # required for integration weights (not sure if it should be here)
-            rotation_angle: 0.0
             # required for mixers (not sure if it should be here)
             mixer_drive_g: 0.0
             mixer_drive_phi: 0.0
             mixer_readout_g: 0.0
             mixer_readout_phi: 0.0
+            # required for integration weights (not sure if it should be here)
+            rotation_angle: 0.0
```

Comment:
This can be removed also in other parts of the runcard.

Reply:
removed